pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -10,9 +10,9 @@
 
 def get_pokemon_ids_from_inventory(response_dict):
     id_list = []
-    inventory_items = response_dict.get("responses", {}).get("GET_INVENTORY", {}).get("inventory_delta", {}).get(
-        "inventory_items")
     if response_dict is not None:
+        inventory_items = response_dict.get("responses", {}).get("GET_INVENTORY", {}).get("inventory_delta", {}).get(
+            "inventory_items")
```

Comment:
Should we handle `KeyError` here? Doing chained `get` calls is kind of pointless if we're assuming that a non-`None` `response_dict` will always have those fields anyway.